File: lesson_22_java/src/main/java/com/lesson22java/entities/Customer.java
package com.lesson22java.entities;

import jakarta.persistence.*;
import lombok.Data;

@Entity
@Table(name = "customer")
@Data
public class Customer {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Long id;
    @Column(name = "fio")
    private String fio;
    @Column(name = "age")
    private Integer age;
}

File: lesson_20/src/main/kotlin/lesson20kt/entities/Customer.kt
package lesson20kt.entities

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "customer")
data class Customer(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    var id: Long? = null,

    @Column(name = "fio")
    var fio: String? = null,

    @Column(name = "age")
    var age: Int? = null,
)